scripts/templates/base.js

```diff
@@ -73,34 +73,31 @@ function paramsToDic(location) {
 
 function filter_data(data, search, authority) {
     if (!search || !search.trim()) {
-        search = ''
+        search = '';
     }
-    let authorities = [authority]
     let s = search.toUpperCase().split(' ');
-    if (!authority) {
-        let possible_authorities = ['EPSG', 'ESRI', 'IAU_2015', 'IGNF', 'NKG', 'OGC']
-        authorities = s
-            .map(a => a.split(':')[0])
-            .filter(a => possible_authorities.includes(a));
 
-        s = s.filter(a => !possible_authorities.includes(a))
-            .map(a => {
-                if (possible_authorities.includes(a.split(':')[0])) {
-                    return a.slice(a.indexOf(':') + 1);
-                }
-                return a;
-        });
+    let auth_code = [null, s.length ? s[0] : null];
+    if (s.length == 1 && s[0].split(':').length == 2) {
+        auth_code = s[0].split(':');
     }
 
     let r = data.filter(d => {
         let name = d.name.toUpperCase();
-        let valid = s.reduce((accum, current) => accum && name.includes(current), true);
+        let outlier = s.find(elem => !name.includes(elem));
+        if (outlier == "WGS84") {
```

Comment:
not sure how easily that could be done here, but FYI PROJ has a "fuzzy" comparison logic in https://github.com/OSGeo/PROJ/blob/a3b225fc9e8ae4423449c2862833bf747a4573ad/src/iso19111/metadata.cpp#L1257 that basically strips all non-significant characters, as well as transforming things that looks like years "19xy" to "xy", so that "WGS1984" and "WGS 84" are identified the same

Reply:
Thanks. In this case the search string is split by spaces, and checked independently. The match is valid if all the pieces appear in the name, but in any order. Relaxing too much each match would produce too many false positives.